govcd/access_control.go

```diff
@@ -68,7 +68,11 @@ func (client Client) GetAccessControl(href, entityType, entityName string, heade
 // * The subject (HREF and Type are mandatory)
 // * The access level (one of ReadOnly, Change, FullControl)
 func (client *Client) SetAccessControl(accessControl *types.ControlAccessParams, href, entityType, entityName string, headerValues map[string]string) error {
+	return client.setAccessControlWithMethod(http.MethodPost, accessControl, href, entityType, entityName, headerValues)
+}
 
+// setAccessControlWithMethod is the same as Client.SetAccessControl but allowing passing a different HTTP method
+func (client *Client) setAccessControlWithMethod(httpMethod string, accessControl *types.ControlAccessParams, href, entityType, entityName string, headerValues map[string]string) error {
```

Comment:
A suggestion for increasing clarity, what about expanding:

setAccessControlWithMethod ->
setAccessControlWith**Http**Method?

Maybe it's just me, but I find "WithMethod" too generic :) The name is not short, but, for example, client.QueryWithNotEncodedParams is even longer.

CC: @vbauzysvmware @Didainius @adambarreiro for input.

Reply:
I agree